Source/gs/GSHandler.cpp

```diff
@@ -147,15 +147,18 @@ void CGSHandler::ResetBase()
 	memset(m_pRAM, 0, RAMSIZE);
 	memset(m_pCLUT, 0, CLUTSIZE);
 	m_nPMODE = 0;
-	m_nSMODE2 = 0;
+	m_nSMODE2 = 0x3; // Interlacing with fullframe
 	m_nDISPFB1.heldValue = 0;
 	m_nDISPFB1.value.q = 0;
-	m_nDISPLAY1.heldValue = 0;
-	m_nDISPLAY1.value.q = 0;
+	m_nDISPLAY1.heldValue = 0x0183227c;
+	m_nDISPLAY1.value.d0 = m_nDISPLAY2.heldValue;
+	m_nDISPLAY1.value.d1 = 0x001bf9ff;
+	m_nDISPLAY1.heldValue = m_nDISPLAY1.value.d0;
```

Comment:
Held value can probably stay 0. It's just a temporary value that is used when a game writes to that register.

I think it would be nicer to set the `q` member as a whole instead of setting individual `d` values. ie.: `m_nDISPLAY1.value.q = 0x001bf9ff0183227c`.

Reply:
Fixed and rebased :)